executor/point_get_test.go

```diff
@@ -593,3 +593,39 @@ func (s *testPointGetSuite) TestClusterIndexCBOPointGet(c *C) {
 		res.Check(testkit.Rows(output[i].Res...))
 	}
 }
+
+func (s *testPointGetSuite) TesPointGetReadLock(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("create table point (id int primary key, c int, d varchar(10), unique c_d (c, d))")
+	tk.MustExec("insert point values (1, 1, 'a')")
+	tk.MustExec("insert point values (2, 2, 'b')")
+	tk.MustExec("alter table point read")
```

Comment:
TiDB not support this syntax.

Use `lock tables point read`.

Reply:
Thanks, fix.